api/models/Ticket.js

```diff
@@ -6,19 +6,28 @@ const { Schema } = mongoose;
 const ticketSchema = new Schema({
 	requestor: { type: Schema.ObjectId, ref: 'User', required: true },
 	mentor: { type: Schema.ObjectId, ref: 'User' },
+	contact: { type: String, required: false },
```

Comment:
This name confuses me. What is the difference between contact and requestor? Maybe you should name it contactInfo? I think fields default to required = false, so you may not have to include that explicitly.